src/stdlib_logger.f90

```diff
@@ -308,6 +308,8 @@ subroutine add_log_unit( self, unit, stat )
         integer :: lun
         character(12) :: specifier
         logical :: question
+        integer :: istat
+        character(len=80) :: msgtxt
```

Comment:
80 characters are a bit short, maybe have bit bigger buffer here, even if it is not used?

Reply:
Delete unused msgtxt.